src/datomic_rest_api/rest/gene.clj

```diff
@@ -551,22 +560,126 @@
+;;                         "font-weight: bold"
+;;                         0)
+;;                :taxonomy "c_elegans"}
+;;               :evidence
+;;               (vmap
+;;                :Person_evidence
+;;                (seq
+;;                 (for [person (:phenotype-info/person-evidence holder)]
+;;                   {:class "person"
+;;                    :id (:person/id person)
+;;                    :label (:person/standard-name person)
+;;                    :taxonomy "all"}))
+;;
+;;                :Anatomy_term
+;;                (seq
+;;                 (for [anatomy (:phenotype-info/anatomy-term holder)]
+;;                   (datomic-rest-api.rest.object/pack-obj "anatomy-term" (:phenotype-info.anatomy-term/anatomy-term anatomy))))
+;;
+;;                :Curator_confirmed
+;;                (seq
+;;                 (for [person (:phenotype-info/curator-confirmed holder)]
+;;                   {:class "person"
+;;                    :id (:person/id person)
+;;                    :label (:person/standard-name person)
+;;                    :taxonomy "all"}))
+;;
+;;                :Paper_evidence
+;;                (seq
+;;                 (for [paper (:phenotype-info/paper-evidence holder)]
+;;                   (evidence-paper paper)))
+;;
+;;                :Remark
+;;                (seq
+;;                 (map :phenotype-info.remark/text
+;;                      (:phenotype-info/remark holder))))}))
+;;          
+;;          "RNAi:"
+;;          (if-let [rp (seq (rnai-phenos pid))]
+;;            (for [r rp
+;;                  :let [holder (entity db r)
+;;                          rnai ((if not?
+;;                                  :rnai/_phenotype-not-observed
+;;                                  :rnai/_phenotype)
+;;                                holder)]]
+;;              {:text
+;;               {:class "rnai"
+;;                :id (:rnai/id rnai)
+;;                :label (:rnai/id rnai)
+;;                :style 0
+;;                :taxonomy "c_elegans"}
+;;               :evidence
+;;               (vmap
+;;                :Genotype (:rnai/genotype rnai)
+;;                :Remark (seq (map :phenotype-info.remark/text
+;;                                  (:phenotype-info/remark holder)))
+;;                :Strain (:strain/id (:rnai/strain rnai))
+;;                :Paper (if-let [paper (:rnai/reference rnai)]
+;;                         (evidence-paper paper)))})))}])
+;;     (into {}))))
+;; 
          
 (def-rest-widget phenotypes [gene]
   {
    :name      (name-field gene)
-
-   :phenotype
-   {:data
-    {:Phenotype              (phenotype-table (d/entity-db gene) (:db/id gene) false)
-     :Phenotype_not_observed (phenotype-table (d/entity-db gene) (:db/id gene) true)}
-    :description
-    "The phenotype summary of the gene"}
+   :drives_overexpression
+    {:data
+      {:Phenotype (phenotype-overexpression (d/entity-db gene) (:db/id gene))}
```

Comment:
I will fix this for the next release. Still working on this section of the code.